.circleci/continue_config.yml

```diff
@@ -326,15 +326,16 @@ workflows:
           matrix:
             parameters:
               engine:
-                - snowflake
-                - databricks
-                - redshift
-                - bigquery
-                - clickhouse-cloud
-          filters:
-            branches:
-              only:
-                - main
+                #- snowflake
```

Comment:
Note: i'll uncomment these immediately prior to merging. It's helpful to be able to run the Athena integration tests for this PR